lxd/images.go

```diff
@@ -106,7 +106,7 @@ func unpack(file string, path string) error {
 	if err != nil {
 		shared.Debugf("Unpacking failed")
 		shared.Debugf(string(output))
-		return err
+		return fmt.Errorf("Unpack failed, %s.", err)
```

Comment:
On Wed, Jul 20, 2016 at 10:54:24AM -0700, Sean Christopherson wrote:
> > @@ -106,7 +106,7 @@ func unpack(file string, path string) error {
> >  	if err != nil {
> >  		shared.Debugf("Unpacking failed")
> >  		shared.Debugf(string(output))
> > -		return err
> > +		return fmt.Errorf("Unpack failed, %s.", err)
> 
> Yeah, `err` isn't super helpful, but unfortunately `output` is often massive, like hundreds of lines.  I'll see what type of messages show up if I grab the first ~80 chars of the output.

The last 20 lines or something may also be useful; that's probably
where there error happened. Do we have verbose on where we shouldn't?

> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/lxc/lxd/pull/2222/files/16015f0ab699178f206e4cc9e2c4a4b6e182d976#r71574248